latest_cov_build.py

```diff
@@ -37,19 +32,28 @@
 tag_name = data['tag_name']
 base_url = 'https://github.com/mozilla/geckodriver/releases/download/' + tag_name + '/geckodriver-' + tag_name + '-'
 
+# Latest grcov version
+latest_version = requests.get('https://api.github.com/repos/marco-c/grcov/releases/latest')
```

Comment:
Since this code and the geckodriver code is the same, let's put it in a function which returns the URL.